packages/mui-material/src/MenuItem/MenuItem.d.ts

```diff
@@ -38,7 +38,7 @@ export type MenuItemTypeMap<P = {}, D extends React.ElementType = 'li'> = Extend
      */
     divider?: boolean;
     /**
-     * Use to apply selected styling.
+     * If `true`, the component is selected.
      * @default false
```

Comment:
`selected` prop is not `false` by default in the `MenuItem` component. Am I missing something here?

Reply:
We don't explicitly set it to `false`, so it is undefined by default. But it doesn't seem to matter much.

e.g., the logic for assigning classnames is `selected && 'selected'`, not `selected === false && 'selected'`.

Reply:
Yes, it's not a major issue but in my opinion it would be misleading in the JSDocs. Either we add a default value to follow the convention or remove the `@default` tag.